src/main/java/com/kinbin/controller/GameController.java

```diff
@@ -66,10 +57,32 @@ public void pulseKinbin(HttpServletResponse response) throws IOException {
         response.sendRedirect("/");
     }
 
-    @GetMapping("/addCard")
-    public void addCard(HttpServletResponse response) throws IOException {
+    @GetMapping("/add_card_form")
+    public ModelAndView addCardForm() {
+        return new ModelAndView("add_card_form");
+    }
+
+    @GetMapping("/add_card")
+    public void addCard(HttpServletResponse response,
+                        @RequestParam(name = "column-name") String columnName) throws IOException {
         Card card = new Card(DEFAULT_ID, CardType.STORY);
-        game.addCard(card, REPLENISHMENT);
+        game.addCard(card, columnName);
+        response.sendRedirect("/");
+    }
+
+    @GetMapping("/add_column_form")
+    public ModelAndView addColumnForm() {
+        return new ModelAndView("add_column_form");
+    }
+
+    @GetMapping("/add_column")
+    public void addColumn(HttpServletResponse response,
+                          @RequestParam(name = "column-type") String columnType) throws IOException {
+        if (columnType.equals("replenishment")) {
+            game.getBoard().addColumn(new Replenishment());
+        } else if (columnType.equals("work-stage")) {
+            game.getBoard().addColumn(new WorkStage());
```

Comment:
If the `columnType` is none of this, the system won't add any column, is that the expected behavior?

Reply:
Yes. As other column types aren't implemented yet, I expected to do nothing and implement any error handling in the future...